aws/aws-sdk-js-v3
# chore(deps-dev): bump fast-xml-parser to 4.0.11

### Issue
followup to https://github.com/aws/aws-sdk-js-v3/pull/3989
ref: https://github.com/awslabs/smithy-typescript/pull/599
~~blocked by: https://github.com/NaturalIntelligence/fast-xml-parser/issues/501~~

### Description
Bump fast-xml-parser to 4.0.11

This change removes the dependency `entities`(13 KB Gzipped). The dependency was introduced in https://github.com/aws/aws-sdk-js-v3/pull/2381 to handle the HTML entities(#2362 #2375). However HTML entities isn't the root cause the the linked issues. They can be fixed by handling [basic xml entities](https://github.com/fb55/entities/blob/master/maps/xml.json). So the chunky dependency can be removed. 

Manually tested in SQS with cases in #2362 and #2375;

<details>
  <summary>Code Snippet</summary>

```typescript
import { SQS } from "@aws-sdk/client-sqs";
import md5 from "md5";

const region = "us-west-2";
const client = new SQS({ region });

const QueueName = `test-${Date.now()}`;
const { QueueUrl } = await client.createQueue({ QueueName });

const MessageBody = `&lt;`;
console.log(MessageBody);
const { MD5OfMessageBody } = await client.sendMessage({
  MessageBody,
  QueueUrl,
});

console.log(
  `\nmd5 check for sent message: ${md5(MessageBody) === MD5OfMessageBody}`
);

const response = await client.receiveMessage({ QueueUrl });
console.log(
  `md5 check for received message: ${
    md5(MessageBody) === md5(response.Messages[0].Body)
  }`
);
console.log(
  `\nStrict Equality Comparison: ${MessageBody === response.Messages[0].Body}`
);

console.log(response.Messages[0].Body);

console.log(
  `\nChecksum Comparison: ${MD5OfMessageBody === response.Messages[0].MD5OfBody}`
);

await client.deleteQueue({ QueueUrl });
```


</details>

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
